include/roaring/portability.h

```diff
@@ -27,11 +27,10 @@
 #include <malloc.h> // this should never be needed but there are some reports that it is needed.
 #endif
 
-#if __SIZEOF_LONG_LONG__ != 8
+#if defined(__SIZEOF_LONG_LONG__) && __SIZEOF_LONG_LONG__ != 8
```

Comment:
It's a good idea, especially since CMake makes it so easy to build as 32-bit by accident. I will move the defines around a bit so we don't catch MinGW or Clang on Windows.